Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -1187,6 +1187,52 @@ def preWrite(self, font):
 		}
 
 
+class LookupList(BaseTable):
+	def table(self):
+		for l in self.Lookup:
+			for st in l.SubTable:
+				if isinstance(st, SingleSubst): return "GSUB"
```

Comment:
how about you check if the subtable class name ends with "Subst" or "Pos" instead?